################################################
#####        params for hik2005                       #######
#####        yuanxun@ep-ep.com                              #######
#####        updateTime:  2024.05.01                              #######
################################################

hik2005:
  verision: "v-0.0.1"
  
  odom4mapTopic: "hik2005/odom_map"
  odom4qrmapTopic: "hik2005/odom_qrmap"
  msgTopic: "hik2005/msg"
  useLidar: false  #true:实施阶段使用Lidar采集二维码坐标
  calTrans: false  #true:实施阶段使用Lidar采集二维码坐标
  show_msg: false
  collect_QRcode: true
  port: 1024
  log_dir: "/home/zl/work/ep-camera-hik/src/hik2005/log/"
  cfg_dir: "/home/zl/work/ep-camera-hik/src/hik2005/config/"

  #qrmap2mapTrans: [26.0745, 2.03618, 0, 0, 0, 0.999419, -0.0340756] #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  # qrmap2mapTrans: [11.4943, -0.0305323, 0, 0, 0, 0.999919, 0.0127641] #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  # qrmap2mapTrans: [11.5092, -0.0253609, 0, 0, 0, 0.999919, 0.0127504] #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  # qrmap2mapTrans: [11.4888, -0.0223434, 0, 0, 0, 0.999919, 0.0127218] #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  qrmap2mapTrans: [311.613, -188.321, 0, 0, 0, -0.717352, 0.696711] #(t_x, t_y, t_z, q_x, q_y, q_z, q_w)
  
  